components/UI/modal/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classes from 'components/UI/modal/modal.module.scss';
+import Backdrop from 'components/UI/backdrop';
+
+const Modal = ({ show, modalClosed, children }) => (
```

Comment:
What if we add a default value for `show`?

Reply:
Dont think so we need a default value here, whenever we will be showing the modal, we will be passing the value as `true` (this true is nothing but the state of the modal, whether to show the modal or not) and accordingly the backdrop and transition will take place.

Once we implement this modal in our code, things will be more clear. Right now it's just the modal code.
Prakash will be going ahead and implement it in a feature.